CHANGES.rst

```diff
@@ -0,0 +1,8 @@
+Changelog
+=========
+
+Here you can see the full list of changes between each release.
+
+Version 0.1 (release date TBD)
```

Comment:
Please use `FIXME` instead of `TBD`, it will be easier to find.  (E.g. `pylint` warn about it.)